src/classes/PREQ_PreventPPlanParent_TEST.cls

```diff
@@ -0,0 +1,239 @@
+
+        List<Program_Plan__c> pPlans = UTIL_UnitTestData_TEST.getMultipleTestProgramPlans(2);
+        insert pPlans;
+
+        List<Plan_Requirement__c> prs = UTIL_UnitTestData_TEST.getMultipleTestPlanRequirements(2);
+        prs[0].Program_Plan__c = pPlans[0].Id;
+        insert prs[0];
+
+        Test.startTest();
+        prs[1].Plan_Requirement__c = prs[0].Id;
+        prs[1].Program_Plan__c = pPlans[0].Id;
+        Database.saveResult results = Database.insert(prs[1], false);
+        Test.stopTest();
+
+        //Verify new plan requirement is not saved
+        List<Plan_Requirement__c> assertPReqs = [select ID from Plan_Requirement__c];
+
+        System.assertEquals(true, results.success);
+        System.assertEquals(2, assertPReqs.size());
+    }
+
+    //Make sure no error is thrown when updating a Plan Requirement who is a child of another plan requirement,
+    //linking to a different program plan and the setting of Validate_Program_Plan_for_Nested_PR__c is off
+    @isTest
+    public static void pReqUpdateChildPRWithDifferentPPSettingOff() {
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c(Validate_Program_Plan_for_Nested_PR__c = false));
+
+        List<Program_Plan__c> pPlans = UTIL_UnitTestData_TEST.getMultipleTestProgramPlans(2);
+        insert pPlans;
+
+        List<Plan_Requirement__c> prs = UTIL_UnitTestData_TEST.getMultipleTestPlanRequirements(2);
+        prs[0].Program_Plan__c = pPlans[0].Id;
+        insert prs[0];
+        prs[1].Plan_Requirement__c = prs[0].Id;
+        insert prs[1];
+
+        Test.startTest();
+        prs[1].Program_Plan__c = pPlans[1].Id;
+        Database.saveResult results = Database.update(prs[1], false);
+        Test.stopTest();
+
+        //Verify new plan requirement is not saved
+        List<Plan_Requirement__c> assertPReqs = [SELECT ID,Program_Plan__c
+                                                FROM Plan_Requirement__c
+                                                WHERE ID = :prs[1].Id
+                                                LIMIT 1];
+
+        System.assertEquals(true, results.success);
+        System.assertEquals(pPlans[1].Id, assertPReqs[0].Program_Plan__c);
+    }
+
+    //Make sure no error is thrown when inserting a Plan Requirement who is not a child of another plan requirement,
+    //linking to program plan and the setting of Validate_Program_Plan_for_Nested_PR__c is on
+    @isTest
+    public static void pReqInsertNonChildPRWithPPSettingOn() {
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c(Validate_Program_Plan_for_Nested_PR__c = true));
+
+        List<Program_Plan__c> pPlans = UTIL_UnitTestData_TEST.getMultipleTestProgramPlans(2);
+        insert pPlans;
+
+        List<Plan_Requirement__c> prs = UTIL_UnitTestData_TEST.getMultipleTestPlanRequirements(2);
```

Comment:
I'm not entirely sure how lines 221-223 are different than the other test block.